Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  OdometerView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Mikaela Caron on 10/11/23.
+//
+
+import SwiftUI
+
+struct OdometerView: View {
+    @Environment(ActionService.self) var actionService
+    @Environment(\.scenePhase) var scenePhase
+    @Bindable private var viewModel: OdometerViewModel
+    @State private var isShowingEditView = false
+    @State private var isShowingAddView = false
+    @State private var selectedOdometerReading: OdometerReading?
+    
+    init(authenticationViewModel: AuthenticationViewModel) {
+        viewModel = OdometerViewModel(authenticationViewModel: authenticationViewModel)
+    }
+
+    var body: some View {
+        NavigationStack {
+            List {
+                ForEach(viewModel.sortedReadings) { reading in
+                    VStack(alignment: .leading, spacing: 8) {
+                        Text("\(reading.distance) \(reading.unitsAreMetric ? "Km" : "M")")
```

Comment:
I think this should be `km` and `mi` for the units

Reply:
fixed in https://github.com/mikaelacaron/Basic-Car-Maintenance/pull/175/commits/cbc191820bd13eb7b526d30443f15deff3e6a3ed